### ISSUE TITLE
Excessive model reloads when manipulating CN image

### ISSUE THREAD

**Issue by vzsg** [godly-devotion/MochiDiffusion#274]
### Running Latest Version

- [X] I am running the latest version

### Processor

M1 (or later)

### Memory

16GB

### What happened?

1. Select a CN-compatible model.
2. Select a CN model.
3. Press the X button for the CN image multiple times.
4. Notice that the model is reloaded after every click, even though there is no real reason to do so.

<img width="947" alt="image" src="https://github.com/godly-devotion/MochiDiffusion/assets/1783465/473631d3-99e1-46ca-a93b-36ac7a3ca9ab">

There seems to be an unnecessary data dependency between the views and the ImageController.

**Comment by jrittvo** [comment]
At least this can probably be fixed in just Mochi specific code.  Someone just opened an issue about the python pipeline in ml-stable-diffusion over at that repo:

`INFO:  python_coreml_stable_diffusion.coreml_model:  Loading a CoreML model through coremltools triggers compilation every time.  The Swift package we provide uses precompiled Core ML models (.mlmodelc) to avoid compile-on-load.`

**Comment by godly-devotion** [comment]
Yeah I did notice that. Also seems to happen when pressing Apply under Settings.

**Comment by jrittvo** [comment]
With something that was updated in one of the Swift files from ml-stable-diffusion 1.0.0 (that we now depend on) causing intermittent crashes when models load, eliminating these unnecessary reloads becomes even more of an issue . . .

**Comment by godly-devotion** [comment]
I misread the initial report. This hasn't been fixed yet.
